src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -238,6 +246,13 @@ private void styleFetchedListView(CheckListView<CitationRelationItem> listView)
                             libraryTab.clearAndSelect(entry.localEntry());
                         });
                         vContainer.getChildren().add(jumpTo);
+
+                        Button compareButton = IconTheme.JabRefIcons.MERGE_ENTRIES.asButton();
+                        compareButton.setTooltip(new Tooltip(Localization.lang("Compare with duplicate entries")));
+                        compareButton.setOnMouseClicked(event -> {
+                            openPossibleDuplicateEntriesWindow(entry);
```

Comment:
Does this mean that every entry is potentially a duplicate?

Reply:
In the broader context of the code, there’s a `DuplicateCheck` object that checks if an entry in the database has a duplicate. When the comparison button is created, the button is only shown when a potential duplicate is detected. This happens because of the `duplicateCheck.containsDuplicate()` method. If this method detects a duplicate for a given entry, the `compareButton` will allow the user to compare the entries. I'm sorry.I may not understand the point of the question, do I need to make any changes?😅

Reply:
Ah, `CitationRelationItem.isLocal()` returns whether it's a duplicate or not?

Reply:
I think `CitationRelationItem.isLocal()` returns true if entries in `Cites` and `Cited by` listview have been detected there is a possible duplicate entry in local database.